ui/core/core-bs3.scss

```diff
@@ -18,6 +18,85 @@
     }
   }
 
+  h1{
+		font-family: $font-family;
+		font-weight: $font-weight-light;
+		font-size: $font-size-xx-large;
+		line-height: $line-height-heading;
+		margin-top: $spacing-large;
+		margin-right: $spacing-large;
+		margin-bottom: $spacing-none;
+		margin-left: $spacing-large;
```

Comment:
These repeated directives are a great opportunity to define and use a mixin.  You could define a heading mixin that includes the margin, line-height, font-family and font-weight directives, then include it in each tag.